src/Instance/InstanceDetails/InstanceAcquisition/InstanceAcquisition.js

```diff
@@ -1,66 +1,85 @@
+
+import css from './InstanceAcquisition.css';
 
 const InstanceAcquisition = ({ accordionId, instanceId }) => {
   const stripes = useStripes();
-  const { isLoading, instanceAcquisition } = useInstanceAcquisition(instanceId);
-  const controlledAccorion = useControlledAccordion(Boolean(instanceAcquisition?.length));
+  const activeTenant = stripes.okapi.tenant;
+  const centralTenant = stripes.user.user?.consortium?.centralTenantId;
+
+  const {
+    isLoading: isLoadingActiveTenantAcquisition,
+    instanceAcquisition: activeTenantAcquisition,
+  } = useInstanceAcquisition(instanceId, activeTenant);
+  const {
+    isLoading: isLoadingCentralTenantAcquisition,
+    instanceAcquisition: centralTenantAcquisition,
+  } = useInstanceAcquisition(instanceId, centralTenant);
+
+  const controlledAcqAccorion = useControlledAccordion(Boolean(activeTenantAcquisition?.length || centralTenantAcquisition?.length));
+  const controlledActiveTenantAcqAccorion = useControlledAccordion(Boolean(activeTenantAcquisition?.length));
+  const controlledCetralTenantAcqAccorion = useControlledAccordion(Boolean(centralTenantAcquisition?.length));
 
   if (!(stripes.hasInterface('order-lines') &&
     stripes.hasInterface('orders') &&
     stripes.hasInterface('acquisitions-units'))) return null;
 
+  const getTenantAccordionLabel = (tenants, id) => tenants?.find(tenant => tenant.id === id).name;
+
   return (
     <Accordion
       id={accordionId}
       label={<FormattedMessage id="ui-inventory.acquisition" />}
-      {...controlledAccorion}
+      {...controlledAcqAccorion}
     >
-      <MultiColumnList
-        id="list-instance-acquisitions"
-        loading={isLoading}
-        contentData={instanceAcquisition}
-        visibleColumns={visibleColumns}
-        columnMapping={columnMapping}
-        formatter={formatter}
-        interactive={false}
-      />
+      {(isUserInConsortiumMode(stripes) && activeTenant !== centralTenant) ? (
+        <>
+          <Accordion
+            id="active-tenant-acquisition-accordion"
+            label={getTenantAccordionLabel(stripes.user.user.tenants, activeTenant)}
+            className={css.tenantAcquisitionAccordion}
+            {...controlledActiveTenantAcqAccorion}
+          >
+            <TenantAcquisition
+              acquisitions={activeTenantAcquisition}
+              isLoading={isLoadingActiveTenantAcquisition}
+              tenantId={activeTenant}
+            />
+          </Accordion>
+
+
+          <Accordion
+            id="central-tenant-acquisition-accordion"
+            label={getTenantAccordionLabel(stripes.user.user.tenants, centralTenant)}
+            className={css.tenantAcquisitionAccordion}
+            {...controlledCetralTenantAcqAccorion}
+          >
+            <TenantAcquisition
+              acquisitions={centralTenantAcquisition}
+              isLoading={isLoadingCentralTenantAcquisition}
+              tenantId={centralTenant}
+            />
+          </Accordion>
```

Comment:
This one seems to be repetitive, could you try thinking of one of the below -
1. a pattern to render a component within an accordion? - may be a wrapper?
2. or conditionally rendering `TenantAcquisition` component within an accordion?